pkg/server/types.go

```diff
@@ -32,6 +32,7 @@ type NetworkKey struct {
 type Config struct {
 	ProvidersDir             string      `json:"providersDir"`
 	RegistryUrl              string      `json:"registryUrl"`
+	RegistryImage            string      `json:"registryImage"`
```

Comment:
Please add this to the TUI as well to allow users to [edit it](https://github.com/daytonaio/daytona/blob/main/pkg/views/server/configure.go). Also, it should be added to the `daytona server config` [view](https://github.com/daytonaio/daytona/blob/main/pkg/views/server/config.go).